src/RichTextEditor/storybook/RichText.stories.tsx

```diff
@@ -30,9 +31,24 @@ Default.args = {
   maxHeight: '300px',
   outline: true,
 }
+Default.parameters = {
```

Comment:
disabled the tests on these stories since the errors were coming from the plugins inside of lexical and didn't look to be something we could fix on our end